scilpy/utils/streamlines.py

```diff
@@ -190,3 +192,45 @@ def warp_tractogram(streamlines, transfo, deformation_data, source):
             = final_streamline.T
         current_position = max_position
         nb_iteration -= 1
+
+
+def filter_tractogram_data(tractogram, streamline_ids):
+    """ Filter tractogram according to streamline ids and keep
+    the data
+
+    Parameters:
+    -----------
+    tractogram: StatefulTractogram
+        Tractogram containing the data to be filtered
+    streamline_ids: array_like
+        List of streamline ids the data corresponds to
+
+    Returns:
+    --------
+    new_tractogram: Tractogram or StatefulTractogram
+        Returns the same type as the input but only with
+    """
+
+    assert isinstance(tractogram, StatefulTractogram), \
+        ("Tractogram must be StatefulTractogram")
```

Comment:
I think that you can remove this now. We consider people follow the description.